travis-ci/managers/fedora.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+# Run this script from the root of the systemd's git repository
+# or set REPO_ROOT to a correct path.
+#
+# Example execution on Fedora:
+# dnf install docker
+# systemctl start docker
+# export CONT_NAME="my-fancy-container"
+# travis-ci/managers/fedora.sh SETUP RUN CLEANUP
+
+PHASES=(${@:-SETUP RUN CLEANUP})
+FEDORA_RELEASE="${FEDORA_RELEASE:-rawhide}"
+CONT_NAME="${CONT_NAME:-fedora-$FEDORA_RELEASE-$RANDOM}"
```

Comment:
This doesn't seem to be working when `SETUP` and `RUN` are run separately. I got
```
FEDORA_RELEASE=latest ./travis-ci/managers/fedora.sh RUN
Run phase
Error response from daemon: No such container: fedora-latest-15116
```

Reply:
That's somewhat expected, `export CONT_NAME="my-fancy-container"` from my example *fixes* it.